src/main/java/org/veupathdb/service/sequence/model/Config.java

```diff
@@ -0,0 +1,27 @@
+package org.veupathdb.service.sequence.model;
+
+import java.util.*;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import org.veupathdb.service.sequence.model.ReferenceSequenceSpec;
+
+
+public class Config {
```

Comment:
Would it be possible to name this something a little bit more descriptive?  Like `{Something}Config` or something that indicates what this is a config for?